src/runtime/asm_arm64.s

```diff
@@ -15,17 +15,28 @@ TEXT runtime·rt0_go(SB),NOSPLIT,$0
 	MOVW	R0, 8(RSP) // argc
 	MOVD	R1, 16(RSP) // argv
 
-	// create istack out of the given (operating system) stack.
-	// _cgo_init may update stackguard.
+	
 	MOVD	$runtime·g0(SB), g
+	// set the per-goroutine and per-mach "registers"
+	MOVD	$runtime·m0(SB), R0
+	// save m->g0 = g0
+	MOVD	g, m_g0(R0)
+	// save m0 to g0->m
+	MOVD	R0, g_m(g)
+
+	// create istack out of the given (operating system) stack.
 	MOVD	RSP, R7
 	MOVD	$(-64*1024)(R7), R0
 	MOVD	R0, g_stackguard0(g)
 	MOVD	R0, g_stackguard1(g)
 	MOVD	R0, (g_stack+stack_lo)(g)
 	MOVD	R7, (g_stack+stack_hi)(g)
 
+#ifdef GOOS_windows
+	BL		runtime·alloc_tls(SB)
```

Comment:
Too many spaces between BL and runtime alloc_tls?

Reply:
Fixed.